Verve/VerveAdapter/ALVerveMediationAdapter.m

```diff
@@ -216,7 +216,11 @@ - (void)updateConsentWithParameters:(id<MAAdapterParameters>)parameters
         NSNumber *hasUserConsent = parameters.hasUserConsent;
         if ( hasUserConsent )
         {
-            [[HyBidUserDataManager sharedInstance] setIABGDPRConsentString: hasUserConsent.boolValue ? @"1" : @"0"];
+            NSString* verveGDPRConsentString = [[HyBidUserDataManager sharedInstance] getIABGDPRConsentString];
+            if ( !verveGDPRConsentString || [verveGDPRConsentString isEqualToString:@""] )
+            {
+                [[HyBidUserDataManager sharedInstance] setIABGDPRConsentString: hasUserConsent.boolValue ? @"1" : @"0"];
```

Comment:
Wow I'm worried that this might override whatever OneTrust write and manages. Have you confirmed?